apps/multi.c

```diff
@@ -272,6 +273,55 @@ static void apps_flower_start(twin_screen_t *screen, int x, int y, int w, int h)
     twin_window_show(window);
 }
 
+#if defined(CONFIG_LOADER_PNG)
+static void apps_blur(twin_screen_t *screen, int x, int y, int w, int h)
+{
+    twin_pixmap_t *raw_background =
+        twin_pixmap_from_file(ASSET_PATH "tux.png", TWIN_ARGB32);
+    if (!raw_background)
+        return;
```

Comment:
Let's build `apps_blur` all the time even if PNG loader is absent. That is, consider the below:
```c
static void apps_blur(twin_screen_t *screen, int x, int y, int w, int h)
{
    twin_pixmap_t *raw_background = NULL;
#if defined(CONFIG_LOADER_PNG)
    raw_background = twin_pixmap_from_file(ASSET_PATH "tux.png", TWIN_ARGB32);
#endif
    if (!raw_background) /* Fallback to a default pattern */
        raw_background = twin_make_pattern();
```